README.md

```diff
@@ -32,6 +32,42 @@ require 'covermymeds_api'  # not needed in a rails app
 
 ## Getting Started
 
+### Default Client
+
+```ruby
+CoverMyMeds.default_client
+```
+
+This will set up a new client with a default host of
+`https://api.covermymeds.com` and all the default paths. It will look for an API
+key and secret in the `CMM_API_ID` and `CMM_API_SECRET` environment variables
+respectively.
+
+### Rails
+
+In Rails, the default client will also check `Rails.application.secrets` for a
+`cmm_api_id` and `cmm_api_secret` before falling back to the environment
+variables.
+
+The gem also includes a Railtie to allow simple configuration in
+`Rails.application.configure` blocks typically found in environment files like
+`config/environments/development.rb` etc. Usage is similar to the standard
+configuration.
+
+```ruby
+Rails.application.configure do
+  config.covermymeds_api.indicators_path = '/'
+  config.covermymeds_api.indicators_host = 'http://localhost:3000'
```

Comment:
Should we just refer to default_host here?  Will our customers actually be using the endpoint-specific URL overrides?